### DISCUSSION TITLE
Add categories for pinned applications

### DISCUSSION THREAD

**Discussion by PadFoot2008** [Open-Shell/Open-Shell-Menu#652] [Ideas]
Hello Guys, I joined Git Hub recently to suggest a feature for Open-Shell. I think it would be a good idea to add a feature to group the pinned application under various categories like Tools, Office, etc. It would make it quite easier to access favourite applications. I have also enclosed a picture.

![image](https://user-images.githubusercontent.com/79647294/110319073-60770680-8034-11eb-90cf-a8138473980b.png)

**Comment by bonzibudd** [comment]
This is a good idea. You can currently do this on right-hand items, but I understand that might not be the most intuitive solution as it uses sub-menus.

**Reply by PadFoot2008** [reply]
Thanks for replying Bonzibud. It would be great if you can add this feature to left-hand side items just as in the image I have included in my previous post. Right hand side is okay.
